lib/storage.js

```diff
@@ -441,6 +441,42 @@ exports.set = function(key, json, options, callback, retries) {
   });
 };
 
+/**
+ * @summary Write user data sync
+ * @function
+ * @public
+ *
+ * @param {String} key - key
+ * @param {Object} json - json object
+ * @param {Function} callback - callback (error)
+ *
+ * @example
+ * const storage = require('electron-json-storage');
+ *
+ * storage.setSync('foobar', { foo: 'bar' }, function(error) {
```

Comment:
The example should not have a callback if its synchronous

Reply:
Done